lib/marc_links.rb

```diff
@@ -119,19 +119,16 @@ def process_link(field)
       rescue URI::InvalidURIError
         return nil
     end
+
     def link_is_fulltext?(field)
       resource_labels = ["table of contents", "abstract", "description", "sample text"]
-      if field.indicator2 == "2"
-        return false
-      elsif field.indicator2 == "0" or field.indicator2 == "1" or field.indicator2.nil? or field.indicator2.empty?
-        resource_labels.each do |resource_label|
-          return false if "#{field['3']} #{field['z']}".downcase.include?(resource_label)
-        end
-        return true
-      else
-        # this should catch bad indicators
-        return nil
+      return false unless %w[0 1].include?(field.indicator2)
+
+      resource_labels.each do |resource_label|
+        return false if "#{field['3']} #{field['z']}".downcase.include?(resource_label)
```

Comment:
I could make this 
```suggestion
        return false if [field['3'], field['z']].any? { |v| v =~ /(table of contents|abstract|description|sample text)/i}
```
to match what we do in `sirsi_config` and make them even more similar

Reply:
Do you think if we updated this in one place we would do it in the other? I like having the consistency of the approach. 👍

Reply:
I'll add a comment to ensure that we do.